Resources/Lang/en_US.json

```diff
@@ -1005,7 +1005,7 @@
   "TrickyInfoLong": "(Add-ons):\nAs the Tricky, your kills will have a random death reason.",
   "TiredInfoLong": "(Add-ons):\nWhenever Tired kills (or uses kill ability on) someone, alternatively whenever they finish a task, they will temporarily get lower vision & lower speed.",
   "StatueInfoLong": "(Add-ons):\nWhenever many people are near the Statue, the Statue is completely frozen or slowed down depending on the settings.",
-  "EvaderInfoLong": "(Add-ons):\nWhen the Evader gets voted out, there is a chance the Evader will not get ejected. (Chance set by host.)",
+  "EvaderInfoLong": "(Add-ons):\nWhen the Evader gets voted out, there is a chance the Evader will not get ejected. (Chance set by the Host.)",
```

Comment:
Good, just remove the second "Evader in the sentence as it is redundant. Consider using "they".